### ISSUE TITLE
ITs on Travis at Times Fail to Connect to Docker

### ISSUE THREAD

**Issue by original-brownbear** [elastic/logstash#7160]
We have a few failures on Travis that show this line:

```sh
 CLI > logstash-plugin remove without internet connection (linux seccomp wrapper) when other plugins depends on this plugin refuses to remove the plugin and display the plugin that depends on it.
     Failure/Error: @fixture = Fixture.new(__FILE__)
     
     Docker::Error::UnexpectedResponseError:
```

E.g. https://travis-ci.org/elastic/logstash/jobs/233917467#L2563

Tests working with Docker appear to be broken by the internet disabling `C` hack we're using in some tests. The reason they break the Docker API and that in turn even breaks tests not using Docker is that the Docker API has a global singleton Docker container set up as soon as it's `require`d that acts in the background and will err out if the internet is down (I think it does some heartbeating).

This needs to be stabilized, like in https://github.com/elastic/logstash/pull/7157#issuecomment-302637245 I suggest just dockerizing the Logstash service and disabling internet via Docker networking API calls in a clean way.

**Comment by original-brownbear** [comment]
@jsvd fyi :)

**Comment by andrewvc** [comment]
I think that the approach of dockerizing logstash here makes sense for the reasons you've stated.

**Comment by original-brownbear** [comment]
ok let me try and fix this. This is happening at quite a rate on Travis.

**Comment by original-brownbear** [comment]
@andrewvc @jsvd bam low cost fix https://github.com/elastic/logstash/pull/7168 :)

**Comment by original-brownbear** [comment]
That said we obviously should still clean this up to using Docker in another issue I guess :)

**Comment by original-brownbear** [comment]
Fixed by #7168
